config/solr_query_config.yml

```diff
@@ -77,49 +84,3 @@ NSIDC:
   phrase_slop: 1
   boost_query: brokered:false^100 published_date:[NOW-2YEARS/DAY TO NOW/DAY]^15 #boost non-brokered datasets and datasets published in the last 2 years
   boost: product(popularity,query({!type=edismax qf=$qf pf=$pf ps=$ps bq=$bq bf=sum(1,product(tan(div(popularity,8)),50))^55 v=$q boost=})) #scale documents by how well they satisfy the query and their popularity
-ADE:
```

Comment:
I don't think we need this section anymore, do we?

Reply:
Nope!